beacon_node/lighthouse_network/src/rpc/config.rs

```diff
@@ -99,11 +99,9 @@ impl RateLimiterConfig {
     pub const DEFAULT_META_DATA_QUOTA: Quota = Quota::n_every(2, 5);
     pub const DEFAULT_STATUS_QUOTA: Quota = Quota::n_every(5, 15);
     pub const DEFAULT_GOODBYE_QUOTA: Quota = Quota::one_every(10);
-    pub const DEFAULT_BLOCKS_BY_RANGE_QUOTA: Quota =
-        Quota::n_every(methods::MAX_REQUEST_BLOCKS, 10);
+    pub const DEFAULT_BLOCKS_BY_RANGE_QUOTA: Quota = Quota::n_every(1024, 10);
```

Comment:
unclear to me whether this should be set at runtime

Reply:
I'm not super fussed about rate limit quotas being exact across forks. I think we should set the defaults to be the higher value. We could thread the ChainSpec to the rate limit config too but I don't think it is necessary.

Reply:
I agree with pawan. We slightly loosen the constraints to deal with the upcoming fork, I dont think really opens a serious attack vector.
Also `MAX_REQUEST_BLOCKS` is a constant at compile time, so its practically identical here, right?

Although there is a `MAX_REQUEST_BLOCKS_DENEB = 128` which seems to be changed here to `1024`

Reply:
> Also MAX_REQUEST_BLOCKS is a constant at compile time, so its practically identical here, right?
>
> Although there is a MAX_REQUEST_BLOCKS_DENEB = 128 which seems to be changed here to 1024

Yes, this is consensus spec's defined the max request size, which changes across the fork. So I'll leave it at 1024 (the existing value). We may want to tune it down later since the max request size decreases in deneb and since we're just handling more data per slot with the advent of blobs